node/src/truv.js

```diff
@@ -15,206 +14,247 @@ if (!API_CLIENT_ID || !API_SECRET) {
  **/
 const getHeaders = () => {
   return {
-    "Content-Type": "application/json",
-    Accept: "application/json",
-    "X-Access-Client-Id": API_CLIENT_ID,
-    "X-Access-Secret": API_SECRET,
+    'Content-Type': 'application/json',
+    'Accept': 'application/json',
+    'X-Access-Client-Id': API_CLIENT_ID,
+    'X-Access-Secret': API_SECRET,
   };
 };
 
+/**
+ * Create a user
+ * https://docs.truv.com/reference/users_create
+ * @returns The response from Truv
+ */
+const createUser = async () => {
+  console.log('TRUV: Requesting a user from https://prod.truv.com/v1/users/');
+  const bodyObj = {
+    external_user_id: `qs-${uuidv4()}`,
+    first_name: 'John',
+    last_name: 'Johnson',
+    email: 'j.johnson@example.com',
+  };
+  const body = JSON.stringify(bodyObj);
+
+  const responseBody = await sendRequest('users/', { body });
+  return responseBody;
+};
+
+/**
+ * Create a bridge token for a user
+ * https://docs.truv.com/reference/users_tokens
+ * @param {String} user_id
+ * @returns The response from Truv
+ */
+const createUserBridgeToken = async (user_id) => {
+  console.log('TRUV: Requesting user bridge token from https://prod.truv.com/v1/users/{user_id}/tokens');
+  console.log(`TRUV: User ID - ${user_id}`);
+
+  const bodyObj = {
+    product_type: API_PRODUCT_TYPE,
+    client_name: 'Truv Quickstart',
+    tracking_info: '1338-0111-A',
+  };
+
+  if (API_PRODUCT_TYPE === 'pll' || API_PRODUCT_TYPE === 'deposit_switch') {
+    bodyObj.account = {
+      account_number: '16002600',
+      account_type: 'checking',
+      routing_number: '123456789',
+      bank_name: 'TD Bank',
+    };
+
+    if (API_PRODUCT_TYPE === 'pll') {
+      bodyObj.account = { ...bodyObj.account, deposit_type: 'amount', deposit_value: '100' };
+    }
+  }
+
+  const body = JSON.stringify(bodyObj);
+
+  const responseBody = await sendRequest(`users/${user_id}/tokens/`, { body });
+  return responseBody;
+};
+
 /**
  * Retrieves a bridge token from Truv
- * https://docs.truv.com/javascript--nodejs#bridge-tokens_create
- * @return The response from Truv - https://docs.truv.com/javascript--nodejs#schemabridgetoken
+ * https://docs.truv.com/reference/bridge-tokens_create
+ * @return The response from Truv
  */
 const getBridgeToken = async () => {
-  console.log(
-    "TRUV: Requesting bridge token from https://prod.truv.com/v1/bridge-tokens"
-  );
+  console.log('TRUV: Requesting bridge token from https://prod.truv.com/v1/bridge-tokens');
```

Comment:
let's remove this method if it not used anymore?

Reply:
Okay